README.md

```diff
@@ -55,6 +55,32 @@ adguardhome-sync run
 adguardhome-sync run --cron "*/10 * * * *"
 ```
 
+
```

Comment:
@techroy23 thank you very much for your contribution.

Could you please add subtitles in the run section, as tere are now two snippets?

### Linux/Mac

### Windows

Reply:
Got it i will create a new PR